### PR TITLE
feat: add Web3.Storage and Estuary to default pinning provider list

### PR DESCRIPTION
**Pull request by SgtPooki**
- feat: add Web3.Storage default pinning provider
- feat: add Estuary default pinning provider

image

### CODE REVIEW

**Comment by SgtPooki** [comment]
Links to compliance reports are all working as well
image

File: src/constants/pinning.js [hunk 1]
```diff
@@ -29,7 +29,20 @@ const pinningServiceTemplates = [
     icon: 'https://dweb.link/ipfs/QmWBaeu6y1zEcKbsEqCuhuDHPL3W8pZouCPdafMCRCSUWk?filename=filebase.png',
     apiEndpoint: 'https://api.filebase.io/v1/ipfs',
     visitServiceUrl: 'https://docs.filebase.com/api-documentation/ipfs-pinning-service-api'
+  },
+  {
+    name: 'Web3.Storage',
+    icon: 'https://bafybeiaqsdwuwemchbofzok4cq7cuvotfs6bgickxdqr6f7hdt7a64cwwa.ipfs.w3s.link/Web3.Storage-logo.svg',
```
Comment (lidel) [inline src/constants/pinning.js:35]:
```suggestion
    icon: 'https://dweb.link/ipfs/bafybeiaqsdwuwemchbofzok4cq7cuvotfs6bgickxdqr6f7hdt7a64cwwa/Web3.Storage-logo.svg',
```

**Review by lidel — CHANGES_REQUESTED** [review]
lgtm, but this will ship with IPFS Desktop and Kubo, it may introduce issue triage load coming from less technical users.

Before we merge this, would like to get official :+1:  from projects themselves.
Pinging:

- web3.storage @alanshaw @vasco-santos or  @olizilla 
- estuary @jimmylee  @whyrusleeping

**Comment by jimmylee** [comment]
LGTM

**Review by vasco-santos — APPROVED** [review]
LGTM 👍🏼

**Review by lidel — APPROVED** [review]
We got :+1:  from both teams :)

**Pull request merged**